src/components/Playhead/Playhead.scss

```diff
@@ -5,7 +5,7 @@
   bottom: 0;
   margin-top: -2px;
   margin-bottom: -2px;
-  pointer-events: none;
+  pointer-events: all;
```

Comment:
This doesn't work in IE, default needs to be set to `visible` to work

Reply:
added no we don't need all, I've checked it.